packages/launcher/src/main/rpc/app.js

```diff
@@ -53,6 +51,15 @@ const getStorageManifestHash = async (
   }
 }
 
+const raiseErrorIfKeyInvalid = (key: string): void | Error => {
+  const validFilename = /^([A-Za-z0-9_. =+-]+?)$/
```

Comment:
Rather than redefining this regex in every function call, you could set it at the module level so it's only defined once.

Reply:
Good point :+1: